hs_access_control/models/user.py

```diff
@@ -2182,7 +2186,9 @@ def can_undo_share_resource_with_user(self, this_resource, this_user):
         if not self.user.is_active:
             raise PermissionDenied("Requesting user is not active")
         if not this_user.is_active:
-            raise PermissionDenied("User is not active")
+            # This causes problems with display of resources
+            # raise PermissionDenied("User is not active")
+            return False
```

Comment:
@alvacouch Line 2186 appears to overshadow line 2188 to 2191. Should lines 2186 and 2187 be removed?

Reply:
Self.user and this_user are different. 
* self.user is the requesting user.  This can't be inactive and an exception is warranted. 
* this_user is the affected user.  This can be inactive and an exception is not warranted.